src/Workspaces/VisualBasic/Portable/CodeStyle/VisualBasicCodeStyleOptionsProvider.vb

```diff
@@ -18,7 +18,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeStyle
 
         Public ReadOnly Property Options As ImmutableArray(Of IOption) Implements IOptionProvider.Options
             Get
-                Return VisualBasicCodeStyleOptions.AllOptions
+                Return VisualBasicCodeStyleOptions.AllOptions.As(Of IOption)
```

Comment:
❔ Do you know if this method converts the underlying array?

Reply:
Yes, it performs the implicit conversion for each element